projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-send-email-preview.php

```diff
@@ -73,20 +74,33 @@ public function permissions_check( $request ) {
 			);
 		}
 
-		$request['context'] = 'edit';
+		$post = get_post( $request['id'] );
```

Comment:
Just a minor suggestion. Is the `id` a request param? Maybe you can use the `get_param()` method.

```suggestion
		$post = get_post( $request->get_param( 'id' ) );
```